### PR TITLE
Added privatemethods in OOP

### PR DESCRIPTION
**Pull request by sabiha22**

### CODE REVIEW

File: 5 Object Oriented Programming/8 privateMethods.js [hunk 1]
```diff
@@ -3,4 +3,36 @@
...
+  ob.a = 10; //Can be accessed directly
+  ob.b = 20; // Can be accessed directly
+  ob.c = 100; // ERROR!
+  ob.setc(100);
+  console.log('a, b, c: ' + ob.a + ' ' + ob.b + ' ' + ob.getc());
+  print(); //ERROR!
+};
+
+/* When a member is private, it cannot be accessed by the members which are outside the class.To make methods, getter/setters or fields private, just give them a name starting with #.
+Public members can be accessed anywhere, even outside the class. All members are public by default.*/
```
Comment (MadaZZ) [inline 5 Object Oriented Programming/8 privateMethods.js:38]: Well explained! Good Job. 👍🏻

**Review by MadaZZ — APPROVED** [review]
You can add the proper header to the file. Check [SampleJS.md](https://github.com/MadaZZ/JS-Concepts/blob/master/SampleJS.md)

**Comment by MadaZZ** [comment]
Well done! 👍🏻

File: 5 Object Oriented Programming/8 privateMethods.js (hunk 1 above)
Comment (sabiha22) [reply 5 Object Oriented Programming/8 privateMethods.js:38]: Thanks :)

**Pull request merged**